src/PiWeb.Import.Sdk/Modules/ImportFormat/IImportFormat.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Zeiss.PiWeb.Import.Sdk.Modules.ImportFormat;
+
+/// <summary>
+/// Represents a custom import format provided as part of a plugin. Custom import formats provide a way to identify the relevant files,
+/// parse the data in those files and upload the data to PiWeb Server. The custom import formats can be configured like built-in formats
```

Comment:
Ohne the, da es um das generelle Konzept geht.

Reply:
Hab ich geändert.